lib/tests.nix

```diff
@@ -61,11 +61,14 @@ let
               "mkTestDerivationFromNixvimModule: the `dontRun` argument is deprecated. You should use the `test.runNvim` module option instead."
               { config.test.runNvim = !dontRun; }
           ))
-          { wrapRc = true; }
+          {
+            wrapRc = true;
+            nixpkgs = {
+              inherit pkgs;
+            };
```

Comment:
Style:

```suggestion
            nixpkgs.pkgs = pkgs;
```

??

Reply:
I generally think of it as cleaner to flatten where you can and only break it up when it gets too long. (or obviously when the parent attribute is used again)